pkcs12.go

```diff
@@ -66,12 +66,13 @@ type encryptedPrivateKeyInfo struct {
 func (i encryptedPrivateKeyInfo) GetAlgorithm() pkix.AlgorithmIdentifier { return i.AlgorithmIdentifier }
 func (i encryptedPrivateKeyInfo) GetData() []byte                        { return i.EncryptedData }
 
-// Error indicating that the input is in an unsupported format. The format may be allowed by the RFC's, but may not be implemented by this package.
+// UnsupportedFormat Error indicates that the input is in an unsupported format.
+// The format may be allowed by the RFC's, but may not be implemented by this package.
 type UnsupportedFormat string
```

Comment:
@nathany that's probably fine. We also don't have any tagged releases yet, so it's still ok to break. We may apply the backwards-compatibility among minor releases once we have a tagged version.